iio-private.h

```diff
@@ -206,6 +206,12 @@ struct iio_context_info {
 	char *uri;
 };
 
+struct iio_scan_block {
```

Comment:
You could have this directly in scan.c, I think it won't be used by any other .c file.